### PR TITLE
Allow Duplicate Saved PDF

### PR DESCRIPTION
**Pull request by ChemiKyle**
Recreation of #17 with removal of commented out debug statements, whitespace adjustment, and variable name retention.

### CODE REVIEW

**Comment by remifrazierucsf** [comment]
@ChemiKyle doesn't this overload `$index` in a way that breaks flow?

File: ExternalModule.php [hunk 1]
```diff
@@ -28,74 +28,85 @@ function hook_survey_complete($project_id, $record, $instrument, $event_id, $gro
...
 
-      $matches = array();
-      $index = 1;
-      $target_upload_field_name = $target_upload_field;
-      if (preg_match('#^(.+)_([0-9]+)$#', $target_upload_field, $matches)) {
-          $target_upload_field_name = $matches[1];
-          $index = (int)$matches[2];
-      }
+      // Iterate along dictionary of targeted keys
+      for($target = 0; $target <= $target_cnt-1; $target++) {
```
Comment (jrpence) [inline ExternalModule.php]: maybe instead use 
`foreach ( $indices as $target=>$index) {`

Comment (ChemiKyle) [reply ExternalModule.php]: Addressed in 9d602b4

**Review by jrpence — APPROVED** [review]
It look good to me.

**Pull request merged**